mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -345,10 +403,12 @@ CLASS ail net/minecraft/entity/Entity
 	METHOD bQ getServer ()Lnet/minecraft/server/MinecraftServer;
 	METHOD bR isImmuneToExplosion ()Z
 	METHOD bS entityDataRequiresOperator ()Z
+	METHOD bT teleportRequested ()Z
 	METHOD bU getPrimaryPassenger ()Lail;
 	METHOD bV getPassengerList ()Ljava/util/List;
 	METHOD bW getPassengersDeep ()Ljava/util/Collection;
-	METHOD bY getTopmostVehicle ()Lail;
+	METHOD bX hasPlayerMount ()Z
```

Comment:
Sorry, I was not clear enough. `hasPlayerJockey` was viable, but this implies the entity can be riding the player. I was just noting that `jockey` implied control, but this does not check that this entity can be controlled by the rider. Now that I think about it, `hasPlayerRider` seems like an adequate name.

Reply:
Ah, I see what you mean now. I can change it to `hasPlayerRider` if you would prefer. That seems almost an equivalent to what's currently there.

Reply:
Mount is wrong IMO, it implies the player is ridden by the entity, when it is the opposite.

Reply:
Lé done. Ì hope `hasRider` is fine. I only saw that I left out the `Player` after I pushed.

Reply:
The `player` part is important in my opinion, but at least it is not wrong anymore

Reply:
Eeyup. Added the `player` already.